logstash-core/lib/logstash/output_delegator_strategies/legacy.rb

```diff
@@ -2,7 +2,7 @@
 module LogStash module OutputDelegatorStrategies class Legacy
   attr_reader :worker_count, :workers
   
-  def initialize(logger, klass, metric, plugin_args={})
+  def initialize(logger, klass, metric, plugin_args)
```

Comment:
There was no need for defaults, and having them led to this bug in the other strategies